_posts/2020-06-15-pulp-2-enters-maintenance-mode.md

```diff
@@ -0,0 +1,30 @@
+---                                                                                                                    
+title: Pulp 2 Enters Maintenance Mode
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+In 2013, the release of Pulp 2 made huge strides in creating an extendable framework for managing several content types. Over the last number of years, the Pulp team have examined user deployments to continue improving Pulp's capabilities as a content management platform. This led to the release of Pulp 3 in December 2019, which includes a multitude of new features, performance improvements, and increased flexibility. For more information on the differences between Pulp 2 and Pulp 3, see [About Pulp 3](https://pulpproject.org/about-pulp-3/).
+
+The Pulp team's main focus is to continue developing the best open source repository management platform available. This means prioritizing the Pulp 3 road map. At this point, apart from critical security issues, there will be no further feature enhancements or bug fixes for Pulp 2. Much of the technical debt of the Pulp 2 project and many of the feature requests has been or will be implemented in Pulp 3.
```

Comment:
s/apart from critical security issues/apart from critical bug fixes and security issues